examples/simplify-MAPbI3-scan-lebesgue/simplify_example/simplify.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
Could you add the filename after the following line, so we do not need to manually check this file.
https://github.com/deepmodeling/dpgen/blob/db6c8f111905481659720dd236f419e33308fcf3/tests/test_check_examples.py#L30